src/neuroglancer/datasource/precomputed/README.md

```diff
@@ -171,7 +171,7 @@ The actual storage of the manifest and mesh fragment data depends on whether the
 
 The `info` file is a JSON-format text file.  The root value must be a JSON object with the following
 members:
-- `"@type"`: Must be `"neuroglancer_multilod_draco"`.
+- `"@type"`: Must be `"neuroglancer_multilod_draco"` or `"neuroglancer_legacy_mesh"`.
```

Comment:
This shouldn't be mentioned here, since this documents the multiscale mesh format.

Reply:
Oops! Done.